cookbook_se/Makefile

```diff
@@ -9,11 +9,11 @@ register_sandbox: docker_build serialize_sandbox
 	flyte-cli register-files -i -p ${PROJECT} -d development -v ${VERSION} -h localhost:30081 ${CURDIR}/_pb_output/*
 
 .PHONY: serialize_sandbox
-serialize_sandbox: docker_build
+serialize_sandbox: dev_docker_build
```

Comment:
i think we need documentation about when to use serialize sandbox.

Reply:
@kumare3 we already have it https://github.com/lyft/flytesnacks/blob/master/cookbook/recipes/core/remote_flyte/README.rst#in-container-serialization